### DISCUSSION TITLE
Feature Request: Sharing dashboards with other users

### DISCUSSION THREAD

**Discussion by xRealNeon** [umami-software/umami#908] [Ideas]
Hey, can you add the functionality to share a dashboard or website with other users.
So that I can edit the website and the others can view the stats of the website.

**Comment by mikecao** [comment]
That already exists, https://umami.is/docs/enable-share-url

**Comment by xRealNeon** [comment]
I mean to share it with other logged-in users, not all that have the link.

**Comment by mikecao** [comment]
This has been discussed before #230. Basically the share URL does everything you need without needing a complex role/permission system.
